compose/foundation/foundation/src/wasmJsTest/kotlin/androidx.compose.foundation.text.selection/WasmSelectionTests.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation.text.selection
+
+import androidx.compose.foundation.text.KeyMapping
+import androidx.compose.foundation.text.createPlatformDefaultKeyMapping
+import org.jetbrains.skiko.hostOs
+import kotlin.test.Test
+import org.jetbrains.skiko.OS
+
+private val ResolvedKeyboardActions
+    get() = when (hostOs) {
+        OS.MacOS -> MacosKeyboardActions
+        else -> DefaultKeyboardActions
+    }
+
+internal class WasmSelectionTests :
```

Comment:
Probably doesn't need to be `internal`. But I checked the tests run anyway - wasn't sure.

Reply:
it's internal, becasue CommonSelectionTests is internal, so we can either:
 * make CommonSelectionTests tests public as well 
 or
 * leave the way it is
 
 both is fine with me, the "leave it as it is option" however is - I can not argue - more fragile - I can imagine that something will change in tooling, internal tests will be just ignored and we'll miss it.